extensions/wikia/Wall/WallExternalController.class.php

```diff
@@ -309,7 +331,7 @@ public function deleteMessage() {
 					// TODO: log/save data
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)